### PR TITLE
Misc edits to new site

### PR DESCRIPTION
**Pull request by arichiv-usds**

### CODE REVIEW

File: _pages/index.md [hunk 1]
```diff
@@ -23,20 +23,20 @@ graphics:
       src: /assets/images/arrow-right-white.png
       alt: 'arrow right'
     title: Make us better.
-    description: We've drafted documentation in conjunction with the open source community so the DoD can participate in the open source and free software communities, even where we may not have copyright. You can help to continue making our open source strategy better. Open an issue or a pull request with your suggestions on how to continue improving these documents.
```
Comment (jordangov) [inline _pages/index.md]: Why take out that we did this in conjunction with the OSS community? That was an important step in the project.

File: _pages/index.md [hunk 2]
```diff
@@ -23,20 +23,20 @@ graphics:
...
+Code.mil is an experiment in open source at the U.S. Department of Defense. The goal is to foster open collaboration with the developer community across the world on DoD open source projects. During the first phase, which launched in early 2017, we wrote a strategy for open sourcing code written by DoD employees, and then we called upon the OSS developer community to help us finalize that plan. You can read the official [press release](https://www.defense.gov/News/News-Releases/News-Release-View/Article/1092364/dod-announces-the-launch-of-codemil-an-experiment-in-open-source/) on the DoD web site.
 
-Our current phase finalized that strategy and involves getting our first open source projects published in the open. We are also calling on the entire DoD developer community to reach out to DDS to begin tracking all of the great OSS projects coming out.
+At the moment we are finalizing that strategy and getting our first open source projects published in the open. We are also calling on the entire DoD developer community to reach out to DDS to begin tracking all of the great OSS projects coming out.
```
Comment (jordangov) [inline _pages/index.md]: The strategy is finalized, I don't think we want to suggest that it isn't ready to go.

File: _pages/frequently-asked-questions.md [hunk 3]
```diff
@@ -9,7 +9,9 @@ subnav:
     href: "/frequently-asked-questions.html#what-is-codemil"
 ---
 
-# What is Code.mil?
+# Frequently Asked Questions
+
+## What is Code.mil?
```
Comment (jordangov) [inline _pages/frequently-asked-questions.md]: Changing these to h2 elements (versus h1) will require a change in `assets/main.js` which dynamically builds the sidenav.

File: _pages/index.md (hunk 1 above)
Comment (arichiv-usds) [reply _pages/index.md]: The phrasing was kind of awkward, I can re-work

Comment (jordangov) [reply _pages/index.md]: 👍

**Comment by jordangov** [comment]
THANK YOU!!

**Pull request merged**